_posts/2021-09-07-design-of-the-koji-room-koji-muro.md

```diff
@@ -0,0 +1,461 @@
+| Wheat bran & rice husk koji (1:1)      |                      50-55 |                                  9.2 |                                      158 |                                1990-2620 |
+| Wheat bran koji                        |                         50 |                                  4.4 |                                      138 |                                1800-2100 |
+| Miso rice koji (polishing ratio 90%)   |                      40-45 |                                  2.5 |                                41.8-64.9 |                                  460-750 |
+| Seishu rice koji (polishing ratio 75%) |                         35 |                                  2.2 |                                       30 |                                  380-440 |
+| Shoyu koji (soybean & wheat 1:1)       |                      40-45 |                                  2.9 |                                      130 |                                1800-2300 |
+
+
+<div class="callout">
+💡 Seishu (sake) rice koji generates the least amount of heat among koji types, for a simple reason: it's the least nutritious. The more you polish rice, the more you strip away lipids, proteins, and minerals which are concentrated in the outer layers. Highly polished rice is little more than a ball of starch with a bit of protein and minerals - so the koji grows slowly. By comparison - shoyu koji, which contains whole soybeans and wheat, is choc full of nutrition and generates over 4x the heat of seishu koji. 
+</div>
+
+
+With these parameters, the remainder of the process is fairly simple: all we have to do is go through four design calculations and determine, in order, the room dimensions, insulation, ventilation, and heating. 
+
+### 1. Dimensions 
+
+Let's determine the size of our table (*toko*) and koji room. To fit 300kg of steamed rice for inoculation (*tane-kiri*) and at the end of the process (*shimai shigoto*) at a depth of 5cm, we need a 13.2 square meter table, housed in a koji room measuring about **8m(L) x 5m (W) x 2.2m(H)** in order to fit clearances and other equipment (See Appendix - Step 1, 2).
+
+### 2. Insulation
+
+The koji room's insulation must be sufficient to keep the room warm AND prevent condensation on the inside surfaces of the room.  If we look at a section of the koji-room wall, overlaid with a steady-state temperature profile, it will look something like this:
+
+![](/assets/images/koji_room/Wallheattransfer.png){:height="550" .center}
+*Depiction of 1-D steady state heat transfer through a wall*
+
+Our koji room, sitting at 30°C and 70% RH, will be slightly colder near the inside wall in an area known as the **temperature boundary layer**. The temperature right up against the interior wall *must* be greater than the dew point of the air in the koji room (which is 23.9°C) or condensation will form. 
```

Comment:
Where is 23.9°C from?

Reply:
Added a brief description. I didn't want to explain psychrometrics in this post or things would get clunky. I may have to on the next post